android/src/main/kotlin/com/airship/flutter/InboxMessageViewFactory.kt

```diff
@@ -1,17 +0,0 @@
-package com.airship.flutter
-
-import android.content.Context
-import io.flutter.plugin.common.BinaryMessenger
-import io.flutter.plugin.common.MethodChannel
-import io.flutter.plugin.common.StandardMessageCodec
-import io.flutter.plugin.platform.PlatformView
-import io.flutter.plugin.platform.PlatformViewFactory
-
-class InboxMessageViewFactory(private val binaryMessenger: BinaryMessenger) : PlatformViewFactory(StandardMessageCodec.INSTANCE) {
```

Comment:
Why was this removed?

Reply:
I have reformat on save, just spacing

Reply:
I just consolidated these the same way the embedded view was consolidated